packages/quantic/force-app/main/default/lwc/subjectInput/subjectInput.html

```diff
@@ -0,0 +1,13 @@
+<template>
+  <h2 class="slds-text-heading_small slds-m-bottom_x-small">
+    {label}
+  </h2>
+  <div class="slds-is-relative">
+    <div class="slds-form-element">
+      <div class="slds-form-element__control">
+        <input value={value} type="text" class="slds-input" oninput={handleChange} maxlength={maxLength} />
```

Comment:
It would be nice to also expose a `required` property. I imagine this field will be mandatory most of the time.